db/migrate/20200406130817_rename_json_columns.rb

```diff
@@ -0,0 +1,42 @@
+class RenameJsonColumns < ActiveRecord::Migration[5.2]
```

Comment:
We'll briefly have the old app running with the new database (unless we deploy with a hard restart), will the old app using `next unless column.sql_type == "json"` mean some data won't be saved to the database between this migration running and the old app processes being killed?

Reply:
Hmm that's a good point, perhaps we should deploy the `next unless column.sql_type == "json"` work in a separate PR first before actually switching over the columns.

Reply:
What would happen to the old app in that case - would it save data in one column but try to read it from another?

Reply:
I think it should be fine, that line `next unless column.sql_type == "json"` is only used to automatically symbolise the keys of the `json` (and soon the `jsonb`) columns.

Reply:
Thanks, @barrucadu, I've moved that to a separate PR: https://github.com/alphagov/publishing-api/pull/1771